types/snuownd/snuownd-tests.ts

```diff
@@ -0,0 +1,21 @@
+// Default parser
+SnuOwnd.getParser().render(''); // $ExpectType string
+
+// Table of contents renderer
+SnuOwnd.getParser(SnuOwnd.getTocRenderer()).render(''); // $ExpectType string
+
+// Constructing a custom parser with totally fresh callbacks and state
+const customCallbacks = SnuOwnd.createCustomCallbacks({
+	blockcode(out, text, language, context) {
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/snuownd/snuownd-tests.ts;
  Line: 9;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
[33c7a61](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66335/commits/33c7a617ad0a07b3c683b97b10a285ad02c2264c)